chemreg/auth/tests/test_authentication.py

```diff
@@ -40,3 +40,11 @@ def test_settings(settings):
         "rest_framework.authentication.BasicAuthentication"
         in settings.REST_FRAMEWORK["DEFAULT_AUTHENTICATION_CLASSES"]
     )
+
+    # Authenticated users can perform any request.
+    # Unauthorised users will only be permitted if the request
+    # method is one of the "safe" methods; GET, HEAD or OPTIONS.
+    assert (
+        "rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnlyDjangoModelPermissionsOrAnonReadOnly"
```

Comment:
```suggestion
        "rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly"
```
hint: doubling the classname does NOT increase security

Reply:
What about tripling it? Thanks for catching that lol